packages/auth/index.extension.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Core functionality shared by all clients
+export * from './src';
+
+// Additional DOM dependend functionality; we need to import and then
+// export separately so that the rollup alias will work (for aliasing these
+// imports in node environments to no-ops and errors... see
+// src/platform_node/index.ts).
+
+// persistence
+import { browserLocalPersistence } from './src/platform_browser/persistence/local_storage';
```

Comment:
I guess we can remove these cos extensions only support indexDB persistence and chrome storage persistence(which we dont support on Auth SDK yet) - https://developer.chrome.com/docs/extensions/mv3/service_workers/service-worker-lifecycle/#persist-data

Reply:
Yeah we'd definitely want to take the opportunity to clean up anything that can't be used in extensions